shared/android/app/src/main/java/io/keybase/ossifrage/util/MobileOSVersion.java

```diff
@@ -0,0 +1,7 @@
+package io.keybase.ossifrage.util;
+
+public final class MobileOSVersion {
+    public static String Get() {
+        return Integer.toString(android.os.Build.VERSION.SDK_INT);
```

Comment:
Can we get this without adding this util?

Reply:
Yes - alternative is adding doing `Integer.toString(android.os.Build.VERSION.SDK_INT)` in both call sites of `initOnce`. So in `KeybasePushNotificationListenerService.java` and `MainActivity.java`. I can change it if that's preferable. Thanks!